bin/auto_hck

```diff
@@ -11,6 +11,8 @@ begin
   require './lib/project'
   require './lib/trap'
 
+  exec File.join(__dir__, 'ns'), __FILE__, *ARGV unless ENV.key?('AUTOHCK_SLIRP')
```

Comment:
From docs: `exec(*args) - Replaces the current process by running the given external command`
We need network namespaces and slirp only for Linux + QEMU. We should have access to the real network in the case of Physical HCK and for Windows (plans). 

So, we need a way to run auto_hck without network namespace.

Reply:
I just dropped the line with commit 65b71aecd7e1a3bd676a1c5cff353bbd331fe9ab. Now you always need to prefix the command line with `bin/ns` if you run QEMU.

Reply:
ok